runner/src/main.rs

```diff
@@ -530,6 +530,8 @@ fn run_example_server(
         spread![
             "--grpc-tls-private-key=./examples/certs/local/local.key".to_string(),
             "--grpc-tls-certificate=./examples/certs/local/local.pem".to_string(),
+            "--http-tls-certificate=./examples/certs/local/local.pem".to_string(),
```

Comment:
(nit) Could you keep these sorted consistently? probably actually reorder the grpc ones